src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,20 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    final String firstLetter = "w";
```

Comment:
move this constant to class please

Reply:
i made it private static final? is it wright? B'cos, how i understand, it was at class